recipes/parsel/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "parsel" %}
+{% set version = "1.5.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4da4262ba4605573b6b72a5f557616a2fc9dee7a47a1efad562752a28d366723
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python
+  run:
+    - cssselect >=0.9
+    - functools32  # [py2k]
```

Comment:
Do you need `py2k`? Otherwise we can make this noarch: python

Reply:
I think this is only a major core dependency of `scrapy`, which is now python 3 only as of version 2. I think it is reasonable to drop `py2k` support, but will wait for one of the other co-maintainers from the project to comment.

Reply:
Or publish at least one build with py2k and make it noarch later.

Reply:
In that case, go ahead and merge assuming  @kmike, @wRAR, and @dangra wants to be maintainers.

Reply:
But making it noarch later is definitely a nice idea!

Reply:
Python 2 being deprecated, even though Parsel supports Python 2 it may make sense to make it noarch already. No strong opinion either way, though. Up to you.